mobile/package.json

```diff
@@ -26,11 +26,14 @@
     "react": "18.0.0",
     "react-dom": "18.0.0",
     "react-native": "0.69.6",
+    "react-native-safe-area-context": "4.3.1",
+    "react-native-screens": "~3.15.0",
     "react-native-web": "~0.18.7",
     "typescript": "^4.6.3"
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
+    "@react-navigation/native": "^6.0.13",
```

Comment:
This should go in the regular dependencies. If you `yarn remove @react-navigation/native` and `yarn add @react-navigation/native` it should fix this.